app/views/merchants/show.html.erb

```diff
@@ -13,10 +13,12 @@
   <% end %>
   </ul>
 </section>
-
+<h3>Items Ready to ship</h3>
 <section class="items_ready_to_ship">
   <% @merchant.items_ready_to_ship.each do |item| %>
-    <li>Item: <%= item.name %> Invoice: <%= link_to "#{item.invoice_id}", "/merchants/#{@merchant.id}/invoices/#{item.invoice_id}" %></li>
+    <strong>Item:</strong> <%= item.name %> 
+    <strong>Date Created:</strong> <%= item.ordered_on_date %> 
+    <strong>Invoice:</strong> <%= link_to "#{item.invoice_id}", "/merchants/#{@merchant.id}/invoices/#{item.invoice_id}" %><br>
```

Comment:
I have not seen the `<strong>` syntax before and I looked it up. Great job using it to bold certain elements on the page!